tools/python_api/src_py/query_result.py

```diff
@@ -73,6 +76,17 @@ def check_for_query_result_close(self):
         if self.is_closed:
             raise Exception("Query result is closed")
 
+    def is_success(self):
```

Comment:
This function should be called in one of the test cases.

Reply:
done